synth.py

```diff
@@ -31,4 +31,6 @@
       destination_name='os-config',
   )
 
-java.common_templates()
\ No newline at end of file
+java.common_templates(excludes=[
```

Comment:
do you need to exclude the readme too?

Reply:
This one is an actual client where we do want to automatically generate the README.

Reply:
After the client goes 1.0.0 and is included in the libraries-bom via google-cloud-bom, we will add back the sample and the README will show the libraries-bom install method.

Reply:
won't this deleted section get regenerated? https://github.com/googleapis/java-os-config/pull/30/files#diff-04c6e90faac2675aa89e2176d2eec7d8L16 or am I missing something?